frontend/src/components/AccountReceiveTable.vue

```diff
@@ -60,12 +60,45 @@
         >.
       </div>
 
-      <div v-if="scanStatus === 'complete'" class="text-caption q-mb-sm">
-        <!-- Show the most recent timestamp and block that were scanned -->
```

Comment:
Another nit, could we make the bottom padding for the sign buttong the same as the same as the top padding?
<img width="462" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/3624aebe-3e45-4c8d-9494-51b676bc5177">

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/685/commits/a72ff26f40e576630de8b78bfc4417ebb4fdfb26

- made the spacing more uniform between the scanning sign verbiage, the sign button, and the most recent announcement data
- updated the advanced setting form vertical spacing
- made the receive header spacing match the send view